parkings/tests/api/operator/test_parking.py

```diff
@@ -390,3 +399,41 @@ def test_enforcement_domain_can_be_specified_on_parking_creation(operator_api_cl
 
     assert not response_parking_data['domain'] == domain.get_default_domain().code
     assert response_parking_data['domain'] == domain.code
+
+
+def test_post_zone_as_string_or_integer(operator_api_client, operator, enforcer, new_parking_data):
+    domain = create_enforced_domain_for_user(operator.user)
+
+    new_parking_data['zone'] = '2'
+    create_payment_zone(code=str(new_parking_data['zone']), domain=domain)
+    response_parking_data = post(operator_api_client, list_url, new_parking_data)
+    assert response_parking_data['zone'] == 2
+
+    new_parking_data['zone'] = 5
+    create_payment_zone(code=str(new_parking_data['zone']), domain=domain)
+    response_parking_data = post(operator_api_client, list_url, new_parking_data)
+    assert response_parking_data['zone'] == 5
+
+    new_parking_data['zone'] = '5A'
+    create_payment_zone(code=str(new_parking_data['zone']), domain=domain)
+    response_parking_data = post(operator_api_client, list_url, new_parking_data)
+    assert response_parking_data['zone'] == '5A'
+
+
+def test_zone_in_correct_domain_is_set_for_parking(
+        operator_api_client, operator, operator_2_api_client, operator_2, new_parking_data
+):
+    domain_1 = create_enforced_domain_for_user(operator.user)
+    domain_2 = create_enforced_domain_for_user(operator_2.user)
```

Comment:
These domain's shouldn't be enforced by the operators.

Operators create the parkings and Enforcers check them. Operators are allowed to create parkings to whichever enforcement domain, but enforcers are allowed to see only the parkings in their own domain.

Reply:
Seems like I had a little misunderstanding about the operator and domain link, this clears it! I’ll check the related code again and fix these 🙂